examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -2193,9 +2193,9 @@ server cluster OtaSoftwareUpdateRequestor = 42 {
   }
 
   struct ProviderLocation {
-    fabric_idx fabricIndex = 0;
     node_id providerNodeID = 1;
     endpoint_no endpoint = 2;
+    fabric_idx fabricIndex = 254;
```

Comment:
lets name it something that highlights it is temporary. Like temporaryNoSpecFabricIndex.

Should it be nullable?  Also need an issue number for its removal.

Reply:
I thought this was some workaround - reading the summary did not make it clear to me that this is really a global index field. I looked (albeit briefly) at the spec and saw the structure with only 2 members. Maybe spec could be updated to also add some "note: fabric scoped" in the struct.

Following an up to date struct now I see "Fabric-scoped data SHALL always include the FabricIndex field to indicate the associated fabric. The FabricIndex field for fabric-scoped data SHALL NOT be 0 or null."

So adding checkmark.